core/network/src/commonMain/kotlin/org/mifospay/core/network/localAssets/MifosLocalAssetDataSource.kt

```diff
@@ -9,47 +9,32 @@
  */
 package org.mifospay.core.network.localAssets
 
+import kotlinx.coroutines.CoroutineDispatcher
+import kotlinx.serialization.json.Json
 import org.mifospay.core.model.City
 import org.mifospay.core.model.Country
 import org.mifospay.core.model.State
-import kotlinx.coroutines.CoroutineDispatcher
-import kotlinx.coroutines.withContext
-import kotlinx.serialization.ExperimentalSerializationApi
-import kotlinx.serialization.json.Json
-import kotlinx.serialization.json.decodeFromStream
 
 class MifosLocalAssetDataSource(
     private val ioDispatcher: CoroutineDispatcher,
     private val networkJson: Json,
     private val assets: LocalAssetManager,
 ) : LocalAssetDataSource {
 
-    @OptIn(ExperimentalSerializationApi::class)
     override suspend fun getCountries(): List<Country> {
-        return withContext(ioDispatcher) {
-            assets.open(COUNTRIES_ASSET).use(networkJson::decodeFromStream)
-        }
+        return emptyList()
```

Comment:
why empty, it should create asset json, same for below

Reply:
Unable to use the file system, getting some compile time error.. I noted it and will be fixed later..and we're not using this, this has been configured for testing but in our project don't have any test classes

Reply:
we are using it while adding the bank account